.github/actions/post-test/action.yml

```
@@ -32,7 +32,7 @@ runs:
           export DD_ENV="ci"
           export DD_SITE="${{ inputs.datadog-site }}"
 
-          timeout 15 npx @datadog/datadog-ci junit upload --service agoric-sdk ./packages/*/junit.xml
+          timeout 300 npx @datadog/datadog-ci junit upload --service agoric-sdk ./packages/*/junit.xml
```

Oh I think the problem is that we're trying to both install and upload in the same timeout. Maybe we should split this up in 2 commands, one under timeout, the other not.

```suggestion
          npx @datadog/datadog-ci --version
          timeout 30 npx @datadog/datadog-ci junit upload --service agoric-sdk ./packages/*/junit.xml
```

---

i would be in favor of removing the timeout all together.
but since i dont know why its here (the commit it was implemented in didn't have a lot of info) i decided to keep it and instead increase it.

if we're keeping the timeout, would 60 seconds suffice?